app/assets/stylesheets/variables/typography.scss

```diff
@@ -232,6 +232,17 @@ $font-book: (
     font-size: 1rem,
     font-weight: 600,
     line-height: 1.4,
+    color: #000
+  ),
+  join-bip-title: (
+    font-size: 2rem,
+    color: white,
+    font-family: Merriweather
+  ),
+  join-bip-text: (
+    font-size: 1.125rem,
+    color: white,
+    font-family: Merriweather
```

Comment:
Think it might be a merge thing, but you have two color definitions for the join-bip-text. Could the select also use $black rather than #000 - cheers :)

Reply:
Think this was just a merge thing. I've fixed this now and will merge this PR.